### PR TITLE
GEOMESA-911 Disjoint attribute index queries do not work with bin format

### PR DESCRIPTION
**Pull request by anthonyccri**
* Merge disjoint attribute queries with the same secondary filter into
  a single scan plan

### CODE REVIEW

**Comment by anthonyccri** [comment]
cc @jnh5y

File: geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/QueryFilterSplitter.scala [hunk 1]
```diff
@@ -217,6 +217,13 @@ class QueryFilterSplitter(sft: SimpleFeatureType) extends Logging {
       // this is a full table scan, we can just append the OR to the secondary filter
       val secondary = orOption(mergeTo.secondary.toSeq ++ andOption(toMerge.primary ++ toMerge.secondary.toSeq))
       mergeTo.copy(secondary = secondary)
+    } else if(toMerge.strategy == StrategyType.ATTRIBUTE && mergeTo.strategy == StrategyType.ATTRIBUTE) {
+      // if we have disjoint attribute queries with the same secondary filter, merge into a multi-range query
```
Comment (elahrvivaz) [inline geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/index/QueryFilterSplitter.scala]: also need to check that the primary filter is on the same attribute.

**Comment by elahrvivaz** [comment]
would be nice to have a new unit test :)

**Comment by elahrvivaz** [comment]
merged as 57c716a9bbe87fd4864dd17098edd40f1b533ce1

**Pull request closed without merging**